IFC4x3/Properties/r/Reference_1mSb87I6b0egBV57bTxz1d/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_1mSb87I6b0egBV57bTxz1d" Name="Reference" UniqueId="70725207-4869-40a2-a2df-14795defd067" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_1mSb87I6b0egBV57bTxz1d" Name="Reference" UniqueId="70725207-4869-40a2-a2df-14795defd067" PrimaryDataType="IfcIdentifier" />
```

Comment:
Was `IfcLabel` also.

Reply:
definately all Identifier in leading edge docs